client/components/ChanStrip.tsx

```diff
@@ -492,7 +492,9 @@ class ChanStrip extends React.PureComponent<
                                         if (
                                             ch.auxLevel[
                                                 this.props.auxSendIndex
-                                            ] >= 0
+                                            ] >= 0 && ch.auxLevel[

```

Comment:
The formatting of the condtional of the if-statement is not easy to read and would benefit a lot from being extracted to a method. (Also, the logic of the conditional is duplicated atleast thrice, given an even bigger reason for a common method.)

Consider creating something like:
`isPositiveNumber(value: number): boolean {
  return value !== null && value >= 0
}`
Then you could call it like this:
`if(isPositiveNumber(ch.auxLevel[this.props.ausSendIndex])) {...}`
Making the code way eaiser to read

Reply:
Fixed the root cause instead. As the impact would be everywhere the auxLevel was used. Both serverside and clientside.